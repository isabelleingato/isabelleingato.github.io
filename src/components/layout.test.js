import React from "react";
import renderer from "react-test-renderer";

import Layout from './layout';

// https://jestjs.io/docs/en/snapshot-testing
// https://reactjs.org/docs/test-renderer.html
// https://jestjs.io/blog/2016/07/27/jest-14.html
// https://jestjs.io/blog/2016/07/27/jest-14.html
describe("Layout", () => {
	it("renders correctly", () => {
		const tree = renderer
			.create(<Layout></Layout>)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});