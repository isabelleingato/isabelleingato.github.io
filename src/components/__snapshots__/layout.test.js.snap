// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div
  style={
    Object {
      "margin": "5vmin",
    }
  }
>
  <header>
    <h1>
      Isabelle Ingato
    </h1>
    <ul
      className="navLinks"
      role="navigation"
    >
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/about/"
        >
          About
        </a>
      </li>
      <li>
        <a
          href="/recommendations/"
        >
          Reviews & Recommendations
        </a>
      </li>
      <li>
        <a
          href="/blog/"
        >
          Blog
        </a>
      </li>
      <li>
        <a
          href="/cssbattles/"
        >
          My CSS Battles Submissions
        </a>
      </li>
    </ul>
  </header>
  <footer>
    Check out the 
    <a
      href="https://github.com/isabelleingato/isabelleingato.github.io"
      target="_blank"
    >
      repo
    </a>
     for this site.
  </footer>
</div>
`;
